support-frontend/assets/pages/[countryGroupId]/checkout.tsx

```diff
@@ -533,7 +549,27 @@ function CheckoutComponent({ geoId }: Props) {
 				.then((response) => response.json())
 				.then((json) => json as StatusResponse);
 
-			processPayment(createSubscriptionResult, geoId);
+			const processPaymentResponse = await processPayment(
+				createSubscriptionResult,
+				geoId,
+			);
+			if (processPaymentResponse.status === 'success') {
+				const order = {
```

Comment:
I think this idea of having a single object in Session storage encapsulating all useful info is much better than having a key/value for each property - it'll be easier to maintain and modify.